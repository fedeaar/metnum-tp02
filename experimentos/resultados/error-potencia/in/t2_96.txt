6.201863000000000000e+06 5.754400000000000000e+04 1.269199000000000000e+06 1.430715000000000000e+06 2.151763000000000000e+06 -3.692839000000000000e+06 7.467970000000000000e+05 2.774280000000000000e+05 -1.416696000000000000e+06 -3.954260000000000000e+05 1.282051000000000000e+06 -6.876500000000000000e+05 9.051100000000000000e+05 -4.806600000000000000e+04 -9.862770000000000000e+05 6.584300000000000000e+04 1.709721000000000000e+06 -1.184360000000000000e+05 7.071540000000000000e+05 1.300383000000000000e+06
5.754400000000000000e+04 4.031189000000000000e+06 -1.160780000000000000e+06 3.051633000000000000e+06 -5.894640000000000000e+05 -1.872876000000000000e+06 1.255918000000000000e+06 5.949560000000000000e+05 1.104855000000000000e+06 7.526930000000000000e+05 -1.294319000000000000e+06 -4.783070000000000000e+05 7.068790000000000000e+05 4.337640000000000000e+05 2.190212000000000000e+06 1.934510000000000000e+05 1.880147000000000000e+06 7.474610000000000000e+05 -3.880640000000000000e+05 -8.661070000000000000e+05
1.269199000000000000e+06 -1.160780000000000000e+06 7.525362000000000000e+06 6.216630000000000000e+05 -1.246000000000000000e+05 -7.350110000000000000e+05 -9.661410000000000000e+05 1.920120000000000000e+06 -2.240878000000000000e+06 2.090060000000000000e+05 -1.162186000000000000e+06 -1.149713000000000000e+06 -1.837410000000000000e+06 1.361394000000000000e+06 -3.029300000000000000e+05 9.002000000000000000e+03 -2.449302000000000000e+06 -8.156150000000000000e+05 -1.271700000000000000e+05 -6.433780000000000000e+05
1.430715000000000000e+06 3.051633000000000000e+06 6.216630000000000000e+05 8.071243000000000000e+06 8.926730000000000000e+05 -1.496767000000000000e+06 6.306670000000000000e+05 2.746520000000000000e+06 1.438117000000000000e+06 2.063118000000000000e+06 -1.577238000000000000e+06 -2.079154000000000000e+06 9.857340000000000000e+05 1.029725000000000000e+06 1.139156000000000000e+06 1.050055000000000000e+06 1.621839000000000000e+06 1.723649000000000000e+06 1.299040000000000000e+05 -2.252682000000000000e+06
2.151763000000000000e+06 -5.894640000000000000e+05 -1.246000000000000000e+05 8.926730000000000000e+05 5.687494000000000000e+06 7.616570000000000000e+05 -2.515190000000000000e+06 5.300300000000000000e+05 -1.537247000000000000e+06 1.948035000000000000e+06 1.818387000000000000e+06 3.295540000000000000e+05 1.367143000000000000e+06 -1.318950000000000000e+06 -1.044523000000000000e+06 2.057561000000000000e+06 8.939970000000000000e+05 -2.243040000000000000e+05 5.330460000000000000e+05 5.994260000000000000e+05
-3.692839000000000000e+06 -1.872876000000000000e+06 -7.350110000000000000e+05 -1.496767000000000000e+06 7.616570000000000000e+05 6.294283000000000000e+06 -3.570130000000000000e+06 -1.185190000000000000e+05 -7.302700000000000000e+05 6.536050000000000000e+05 -2.935500000000000000e+04 8.235520000000000000e+05 -5.130610000000000000e+05 -1.459235000000000000e+06 -1.402024000000000000e+06 8.775090000000000000e+05 -2.470854000000000000e+06 8.079130000000000000e+05 -2.033900000000000000e+04 -1.315176000000000000e+06
7.467970000000000000e+05 1.255918000000000000e+06 -9.661410000000000000e+05 6.306670000000000000e+05 -2.515190000000000000e+06 -3.570130000000000000e+06 8.265581000000000000e+06 -1.115472000000000000e+06 2.748404000000000000e+06 -3.103536000000000000e+06 -1.556293000000000000e+06 -4.956740000000000000e+05 4.304560000000000000e+05 2.435204000000000000e+06 -2.605630000000000000e+05 -1.451289000000000000e+06 1.728444000000000000e+06 -5.286120000000000000e+05 -1.269611000000000000e+06 1.089079000000000000e+06
2.774280000000000000e+05 5.949560000000000000e+05 1.920120000000000000e+06 2.746520000000000000e+06 5.300300000000000000e+05 -1.185190000000000000e+05 -1.115472000000000000e+06 4.896831000000000000e+06 -1.852970000000000000e+05 2.892047000000000000e+06 -1.228747000000000000e+06 6.239100000000000000e+04 7.344110000000000000e+05 -9.770820000000000000e+05 3.417950000000000000e+05 2.269624000000000000e+06 -4.123100000000000000e+05 5.528340000000000000e+05 1.525426000000000000e+06 -2.176010000000000000e+05
-1.416696000000000000e+06 1.104855000000000000e+06 -2.240878000000000000e+06 1.438117000000000000e+06 -1.537247000000000000e+06 -7.302700000000000000e+05 2.748404000000000000e+06 -1.852970000000000000e+05 6.416896000000000000e+06 6.334030000000000000e+05 -2.042149000000000000e+06 4.131610000000000000e+05 1.162465000000000000e+06 -1.307540000000000000e+05 6.528940000000000000e+05 1.971860000000000000e+05 -9.234600000000000000e+04 -1.902142000000000000e+06 -1.216870000000000000e+05 -4.944030000000000000e+05
-3.954260000000000000e+05 7.526930000000000000e+05 2.090060000000000000e+05 2.063118000000000000e+06 1.948035000000000000e+06 6.536050000000000000e+05 -3.103536000000000000e+06 2.892047000000000000e+06 6.334030000000000000e+05 7.791944000000000000e+06 2.567606000000000000e+06 5.627530000000000000e+05 1.516895000000000000e+06 3.966150000000000000e+05 -1.242890000000000000e+06 1.188775000000000000e+06 8.466080000000000000e+05 7.483800000000000000e+05 1.873860000000000000e+05 6.713560000000000000e+05
1.282051000000000000e+06 -1.294319000000000000e+06 -1.162186000000000000e+06 -1.577238000000000000e+06 1.818387000000000000e+06 -2.935500000000000000e+04 -1.556293000000000000e+06 -1.228747000000000000e+06 -2.042149000000000000e+06 2.567606000000000000e+06 5.925718000000000000e+06 5.620910000000000000e+05 -3.234400000000000000e+04 3.462490000000000000e+05 -3.070038000000000000e+06 -2.319720000000000000e+05 -8.943200000000000000e+04 1.122106000000000000e+06 -1.385219000000000000e+06 2.050714000000000000e+06
-6.876500000000000000e+05 -4.783070000000000000e+05 -1.149713000000000000e+06 -2.079154000000000000e+06 3.295540000000000000e+05 8.235520000000000000e+05 -4.956740000000000000e+05 6.239100000000000000e+04 4.131610000000000000e+05 5.627530000000000000e+05 5.620910000000000000e+05 4.972199000000000000e+06 -2.027083000000000000e+06 -9.395070000000000000e+05 2.761630000000000000e+05 -1.830980000000000000e+05 -2.011019000000000000e+06 -8.602700000000000000e+05 -1.380720000000000000e+05 1.741680000000000000e+06
9.051100000000000000e+05 7.068790000000000000e+05 -1.837410000000000000e+06 9.857340000000000000e+05 1.367143000000000000e+06 -5.130610000000000000e+05 4.304560000000000000e+05 7.344110000000000000e+05 1.162465000000000000e+06 1.516895000000000000e+06 -3.234400000000000000e+04 -2.027083000000000000e+06 5.613139000000000000e+06 8.487380000000000000e+05 -5.499200000000000000e+04 2.098736000000000000e+06 3.094211000000000000e+06 -1.204521000000000000e+06 3.653200000000000000e+05 1.754960000000000000e+05
-4.806600000000000000e+04 4.337640000000000000e+05 1.361394000000000000e+06 1.029725000000000000e+06 -1.318950000000000000e+06 -1.459235000000000000e+06 2.435204000000000000e+06 -9.770820000000000000e+05 -1.307540000000000000e+05 3.966150000000000000e+05 3.462490000000000000e+05 -9.395070000000000000e+05 8.487380000000000000e+05 6.485840000000000000e+06 -1.389008000000000000e+06 -2.809801000000000000e+06 6.674680000000000000e+05 -1.327730000000000000e+05 -1.972621000000000000e+06 -2.097716000000000000e+06
-9.862770000000000000e+05 2.190212000000000000e+06 -3.029300000000000000e+05 1.139156000000000000e+06 -1.044523000000000000e+06 -1.402024000000000000e+06 -2.605630000000000000e+05 3.417950000000000000e+05 6.528940000000000000e+05 -1.242890000000000000e+06 -3.070038000000000000e+06 2.761630000000000000e+05 -5.499200000000000000e+04 -1.389008000000000000e+06 6.409656000000000000e+06 1.796243000000000000e+06 1.112317000000000000e+06 4.601580000000000000e+05 8.539730000000000000e+05 -8.259270000000000000e+05
6.584300000000000000e+04 1.934510000000000000e+05 9.002000000000000000e+03 1.050055000000000000e+06 2.057561000000000000e+06 8.775090000000000000e+05 -1.451289000000000000e+06 2.269624000000000000e+06 1.971860000000000000e+05 1.188775000000000000e+06 -2.319720000000000000e+05 -1.830980000000000000e+05 2.098736000000000000e+06 -2.809801000000000000e+06 1.796243000000000000e+06 6.612137000000000000e+06 1.726680000000000000e+05 -1.046724000000000000e+06 -7.520600000000000000e+05 -6.721660000000000000e+05
1.709721000000000000e+06 1.880147000000000000e+06 -2.449302000000000000e+06 1.621839000000000000e+06 8.939970000000000000e+05 -2.470854000000000000e+06 1.728444000000000000e+06 -4.123100000000000000e+05 -9.234600000000000000e+04 8.466080000000000000e+05 -8.943200000000000000e+04 -2.011019000000000000e+06 3.094211000000000000e+06 6.674680000000000000e+05 1.112317000000000000e+06 1.726680000000000000e+05 6.401338000000000000e+06 -6.689520000000000000e+05 -4.193320000000000000e+05 3.330770000000000000e+05
-1.184360000000000000e+05 7.474610000000000000e+05 -8.156150000000000000e+05 1.723649000000000000e+06 -2.243040000000000000e+05 8.079130000000000000e+05 -5.286120000000000000e+05 5.528340000000000000e+05 -1.902142000000000000e+06 7.483800000000000000e+05 1.122106000000000000e+06 -8.602700000000000000e+05 -1.204521000000000000e+06 -1.327730000000000000e+05 4.601580000000000000e+05 -1.046724000000000000e+06 -6.689520000000000000e+05 5.782661000000000000e+06 1.401281000000000000e+06 -5.503800000000000000e+05
7.071540000000000000e+05 -3.880640000000000000e+05 -1.271700000000000000e+05 1.299040000000000000e+05 5.330460000000000000e+05 -2.033900000000000000e+04 -1.269611000000000000e+06 1.525426000000000000e+06 -1.216870000000000000e+05 1.873860000000000000e+05 -1.385219000000000000e+06 -1.380720000000000000e+05 3.653200000000000000e+05 -1.972621000000000000e+06 8.539730000000000000e+05 -7.520600000000000000e+05 -4.193320000000000000e+05 1.401281000000000000e+06 3.918357000000000000e+06 1.681873000000000000e+06
1.300383000000000000e+06 -8.661070000000000000e+05 -6.433780000000000000e+05 -2.252682000000000000e+06 5.994260000000000000e+05 -1.315176000000000000e+06 1.089079000000000000e+06 -2.176010000000000000e+05 -4.944030000000000000e+05 6.713560000000000000e+05 2.050714000000000000e+06 1.741680000000000000e+06 1.754960000000000000e+05 -2.097716000000000000e+06 -8.259270000000000000e+05 -6.721660000000000000e+05 3.330770000000000000e+05 -5.503800000000000000e+05 1.681873000000000000e+06 6.837249000000000000e+06
