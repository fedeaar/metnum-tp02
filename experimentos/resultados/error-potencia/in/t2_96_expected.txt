1.960957801434775814e+07
1.761056776602390036e+07
1.461869405976227485e+07
1.289633788094605319e+07
1.111823747529082559e+07
9.156236933880645782e+06
8.760100934214297682e+06
6.066116090047590435e+06
5.679333612043344416e+06
4.683443419959976338e+06
3.424837291666640900e+06
3.019092658828140702e+06
2.514850242440759204e+06
1.850048541206123307e+06
1.453315652354216669e+06
1.125769969362810254e+06
3.744204384442308219e+05
9.287211470702933730e+04
7.503374697824839677e+04
1.209315749513045375e+04
